test/core/sequencer/sw.js

```diff
@@ -0,0 +1,72 @@
+var sw = require('../../examples/sw');
+var test = require('tape');
+
+function SWInstallation(registration){
```

Comment:
Here, should we instead use the `setupCache()` method from our actual code, rather than make a unique worker management code segment within the tests? Does that make sense? What do you think? Then our assertions below ought to still pass, right?

Reply:
sure sir ,it will avoid code duplication.
And i also thinks it will still pass.☺️

Reply:
@jywarren could you please review these changes ☺️